pkg/backend/display/rows.go

```diff
@@ -435,19 +435,14 @@ func (data *resourceRowData) getInfoColumn() string {
 }
 
 func getDiffInfo(step engine.StepEventMetadata, action apitype.UpdateKind) string {
-	diffOutputs := action == apitype.RefreshUpdate
 	changesBuf := &bytes.Buffer{}
 	if step.Old != nil && step.New != nil {
 		var diff *resource.ObjectDiff
 		// An OpSame might have a diff due to metadata changes (e.g. protect) but we should never print a property diff,
 		// even if the properties appear to have changed. See https://github.com/pulumi/pulumi/issues/15944 for context.
 		if step.Op != deploy.OpSame {
 			if step.DetailedDiff != nil {
-				diff = engine.TranslateDetailedDiff(&step)
-			} else if diffOutputs {
```

Comment:
Do we not need to still have this but behind the flag now?  Or is there some reason I'm missing why this is no longer needed?

Reply:
In theory this should behave fine since `diff` will be `nil` -- I call this out in the use of the flag later on but as you say it's probably worth putting this in the commit message/PR comment too. Will sort that.